### ISSUE TITLE
[Bug] TDS PriceLockup - Component center alignment issues

### ISSUE THREAD

**Issue by tarekamin** [telus/tds-core#825]
## Description

TDS PriceLockup does not centre correctly when wrapped by a centrally aligned component. PriceLockup was implemented in Site Builder - wrapped in a block (BlockPriceLockup). When BlockPriceLockup gets used in a centrally aligned fashion, the top and bottom text get centred while the price, sign, and rate text remain left aligned. This was replicated in the TDS catalogue. 

## Reproduction Steps

- go here https://tds.telus.com/components/index.html#pricelockup
- In the view code section enter this snippet:
```
<FlexGrid>
  <FlexGrid.Row horizontalAlign="center">
    <FlexGrid.Col xs={12} md={12}>
      <PriceLockup
        size="medium"
        topText="Starting at"
        price="25"
        rateText="/month"
        bottomText="$68 /month after 3 months"
        signDirection="left"
      />
    </FlexGrid.Col>
  </FlexGrid.Row>
</FlexGrid>
```

## Meta

- TDS version: v1.0.0
- Willing to develop solution: Willing to get help
- Has workaround: Yes (wrap in left aligned components)
- High impact: No

## Screenshots

![image](https://user-images.githubusercontent.com/29206760/48962130-cfe28e00-ef30-11e8-8816-49c8f06f11ff.png)

**Comment by nicmak** [comment]
@tarekamin  Thanks, I will be looking into this!

**Comment by tarekamin** [comment]
Thanks @nicmak! Let me know if you need further support.

**Comment by theetrain** [comment]
This issue can also be tracked on Jira: https://telusdigital.atlassian.net/browse/TDS-1047

**Comment by jraff** [comment]
@tarekamin fixed with [@tds/core-pricelockup@1.0.3](https://github.com/telus/tds-core/releases/tag/%40tds%2Fcore-price-lockup%401.0.3)

**Comment by tarekamin** [comment]
Thanks so much folks! Verified it behaves correctly on Site Builder.
